src/fides/api/models/property.py

```diff
@@ -214,3 +254,26 @@ def link_experience_configs_to_property(
 
     prop.save(db)
     return prop.experiences
+
+
+class MessagingTemplateToProperty(Base):
+    @declared_attr
+    def __tablename__(self) -> str:
+        return "messaging_template_to_property"
```

Comment:
Do we need another unique constraint on this table for `messaging_template_id` and `property_id`? In other words, restrict so each pair just shows up once

Reply:
Ah this makes sense 👍